pkg/config/piped.go

```diff
@@ -311,6 +316,13 @@ func (r *HelmChartRepository) Validate() error {
 		return nil
 	}
 
+	if r.IsOCIRepository() {
+		if r.Address == "" {
+			return errors.New("address must be set")
+		}
+		return nil
+	}
+
 	return fmt.Errorf("either %s repository or %s repository must be configured", HTTPHelmChartRepository, GITHelmChartRepository)
```

Comment:
this error is not right anymore, should update that.

Reply:
Updated on this commit :pray: → [96af77c](https://github.com/pipe-cd/pipecd/pull/3643/commits/96af77c33203b4f6ef800c664dd5edbb01312373)